pages/date-range-picker/with-default-date.page.tsx

```diff
@@ -1,43 +0,0 @@
-// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
-// SPDX-License-Identifier: Apache-2.0
-import React, { useState } from 'react';
-
-import { Box, DateRangePicker, DateRangePickerProps, FormField, Link } from '~components';
-
-import ScreenshotArea from '../utils/screenshot-area';
-import { i18nStrings, isValid, relativeOptions } from './common';
-
-export default function DatePickerScenario() {
-  const [value, setValue] = useState<DateRangePickerProps['value']>({
-    type: 'absolute',
-    startDate: '2018-01-09T12:34:56Z',
-    endDate: '2018-01-19T15:30:00Z',
-  });
-
-  return (
-    <Box padding="s">
-      <h1>Date range picker with selected date</h1>
-      <Link id="focus-dismiss-helper">Focusable element before the date range picker</Link>
-      <br />
-      <br />
-      <ScreenshotArea>
```

Comment:
If there is a `<ScreenshotArea>`, it is likely used in some screenshot tests. Did you clean them up before merging this?

Reply:
All tests that run against `<...>/index.html#/light/date-range-picker/with-default-date` and `<...>/index.html#/light/date-range-picker/with-default-date-only` will continue to run but this time against the same page`<...>/index.html#/light/date-range-picker/with-value` with params added to load the page with the component as date-only or not`?dateOnly=true` and `?dateOnly=false`. So the same screenshots will exist. However as this PR  is merged, the regression tests will see the former as removed, and those after it as new, even though they are visually the same